lib/wallet/rpc.js

```diff
@@ -1170,38 +1172,187 @@ class RPC extends RPCBase {
   }
 
   async listTransactions(args, help) {
+    throw new Error('Deprecated: `listtransactions`. ' +
+                    'Use `listhistory` and related methods.');
+  }
+
+  async listHistory(args, help) {
```

Comment:
oh funny, i think i have a minimum args PR for this original call, but should be added here still